staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -680,8 +680,10 @@ message LinuxPodSandboxStats {
     NetworkUsage network = 3;
     // Stats pertaining to processes in the pod sandbox.
     ProcessUsage process = 4;
+    // Swap usage gathered for the pod sandbox.
+    SwapUsage swap = 5;
```

Comment:
from https://earthly.dev/blog/backward-and-forward-compatibility/
```
This means once a unique field number or length is set, it cannot be changed. Any program consuming or serializing protobuf data needs the number to be fixed forever, or both the sender and all the receivers must be updated.
```
I think we should make swap = 6